maintainers/maintainer-list.nix

```diff
@@ -10868,12 +10868,6 @@
     githubId = 8798449;
     name = "kidsan";
   };
-  kierdavis = {
```

Comment:
Since you're removing yourself the commit message should be `maintainers: remove kierdavis`